src/Command.js

```diff
@@ -173,6 +176,43 @@ export class Command {
 		})
 	}
 
+	schedule() {
+		return Promise.reject('Command does not implement schedule(): ' + this.name)
+	}
+
+	registerSchedule() {
+		return new Promise((resolve, reject) => {
+			this.schedule().then((scheduler) => {
+				this.success(this.name + ' has been scheduled to run next at: ' + scheduler.nextOccurence())
+				scheduler.start(() => {
+					this.execAsChildProcess().then((output) => {
+						this.info(output)
+					}).catch((err) => {
+						this.error(err)
+					})
+				})
+			})
+		})
+	}
+
+	execAsChildProcess() {
+		const exec = require('child_process').exec
+
+		const options = {
+			env: process.env
+		}
+
+		return new Promise(async (resolve, reject) => {
```

Comment:
Using `async`/await` here is unnecessary given that you’re already returning a promise